### ISSUE TITLE
Feature request: let users send label predictions and choose metric to plot later

### ISSUE THREAD

**Issue by noncuro** [wandb/client#1044]
Hi, I'm a big fan of Weights and Biases. I was just hoping to suggest a feature that would be hugely valuable to myself and my colleagues! I use various classification/regression metrics to monitor models as they train, mainly from sklearn.metrics, but I never know which will be the most useful when I start training. It's really annoying to have to log _every_ metric I can think of with every possible configuration (macro, micro, weighted...) but they're all basically of the form `metric(y_true, y_pred)`. 

**Request:** It would be amazing if I could log `y_true` and `y_pred` at every test step (or just `y_true` once and `y_pred` at every step) and then choose metrics later from the dashboard. Perhaps metrics from sklearn can be exposed as plot expressions? 

If you could find a way to make this possible, that would be amazing!

**Comment by tyomhak** [comment]
Hi there! Thank you very much for the feedback! We will definitely consider to add that feature!

**Comment by cvphelps** [comment]
Hi there, thanks for this suggestion. We do have computed columns in our roadmap— letting you calculate new column values based on columns you log to your runs.

**Comment by ariG23498** [comment]
Hey @noncuro 
In the past year we've majorly reworked the CLI and UI for Weights & Biases. We're closing issues older than 6 months. Please comment to reopen.
